docs/diagnostics/PX1007.md

```diff
@@ -28,14 +36,13 @@ The diagnostic is currently displayed for the following public entities:
  - DAC property fields
 
 The diagnostic is not displayed for the following items: 
- - Entities with the `PXHidden`, `PXInternalUseOnly`, or `Obsolete` attribute.
+ - Entities with the `PXInternalUseOnly` or `Obsolete` attribute.
+   ** Note: ** The `PXHidden` attribute does not suppress the diagnostic.
```

Comment:
This is not correct. `PXHidden` attribute suppresses the XP1007 diagnostic but only for DACs. This attribute is applied only to DACs in general. It shouldn't be used on DAC extensions or DAC properties. The changes in this release ensured that the declaration of `PXHidden` on something other than DAC will have no effect.